spec/03-plugins/26-prometheus/08-unit_spec.lua

```diff
@@ -0,0 +1,131 @@
+
+describe("Plugin: prometheus (unit)", function()
+  local prometheus
+
+  setup(function()
+    ngx.shared = require("spec.fixtures.shm-stub")
```

Comment:
let's revert back ngx.shared and package loaded["prometheus_resty_counter"] in teardown stage

Reply:
`teardown` added.

but we do not need to revert `package_loaded["prometheus_resty_counter"]`.